pkg/printers/internalversion/printers.go

```diff
@@ -1201,22 +1196,24 @@ func getNodeExternalIP(node *api.Node) string {
 	return "<none>"
 }
 
-// findNodeRole returns the role of a given node, or "" if none found.
-// The role is determined by looking in order for:
-// * a node-role.kubernetes.io/master label
-// * a kubernetes.io/role label
-// * a kubeadm.alpha.kubernetes.io/role label
-func findNodeRole(node *api.Node) string {
-	if _, ok := node.Labels[LabelNodeRoleMaster]; ok {
-		return "Master"
-	}
-	if role := node.Labels[NodeLabelRole]; role != "" {
-		return strings.Title(role)
-	}
-	if role := node.Labels[NodeLabelKubeadmAlphaRole]; role != "" {
-		return strings.Title(role)
+// findNodeRoles returns the roles of a given node.
+// The roles are determined by looking for:
+// * a node-role.kubernetes.io/<role>="" label
```

Comment:
The point of that format was to support multiple roles, and to allow things like this to treat roles generically